src/shipment/controller.js

```diff
@@ -225,9 +226,18 @@ async function getOne(req, res) {
       const [_, { allShipments }] = await getPopulatedShipmentData(sid);
       const shipment = allShipments[0];
 
+      let shipmentDollarValue = 0;
+      for ( m of shipment.manifest) {
+        for ( item of m.items) {
+          const { qty, finalUnitRate } = item;
+          shipmentDollarValue = currency(shipmentDollarValue).add( currency(finalUnitRate).multiply(qty) ).value;
```

Comment:
```
let shipmentDollarValue = currency(0);
for..for.. shipmentDollarValue = shipmentDollarValue.add( currency(finalUnitRate).multiply(qty) );
```
then return `shipmentDollarValue.format()`

Reply:
updated to use `.format()` method over getting `.value`